src/components/forms/OpaqueIngressSecretForm.js

```diff
@@ -0,0 +1,425 @@
+          />
+        </IcseFormGroup>
+        <IcseFormGroup>
+          <Tile className="formInSubForm">
+            <IcseFormGroup>Arbitrary Secret</IcseFormGroup>
+            <IcseFormGroup>
+              <IcseTextInput
+                id={composedId + "-arb-secret-name"}
+                componentName={this.props.data.arbitrary_secret_name}
+                labelText="Name"
+                placeholder="my-secret-name"
+                value={this.state.arbitrary_secret_name}
+                onChange={this.handleInputChange}
+                invalidCallback={() =>
+                  this.props.secretCallback(this.state, this.props)
+                }
+                invalidText={this.props.secretCallbackText(
+                  this.state,
+                  this.props
+                )}
+                field="arbitrary_secret_name"
+                className="fieldWidth"
+              />
+              <IcseTextInput
+                id={composedId + "-arb-secret-description"}
+                componentName={this.props.data.arbitrary_secret_description}
+                labelText="Description"
+                placeholder="my-secret-description"
+                value={this.state.arbitrary_secret_description}
+                onChange={this.handleInputChange}
+                invalidCallback={() =>
+                  this.props.descriptionInvalid(this.state, this.props)
+                }
+                invalidText={this.props.descriptionInvalidText(
+                  this.state,
+                  this.props
+                )}
+                field="arbitrary_secret_description"
+                className="fieldWidth"
+              />
+            </IcseFormGroup>
+            <IcseFormGroup>
+              <IcseTextInput
+                id={composedId + "-arb-secret-data"}
+                componentName={this.props.data.arbitrary_secret_data}
+                labelText="Data"
+                placeholder="my-secret-data"
+                value={this.state.arbitrary_secret_data}
+                onChange={this.handleInputChange}
+                invalidCallback={() =>
+                  this.props.descriptionInvalid(this.state, this.props)
+                }
+                invalidText={this.props.descriptionInvalidText(
+                  this.state,
+                  this.props
+                )}
+                field="arbitrary_secret_data"
+                className="fieldWidth"
+              />
+            </IcseFormGroup>
+            <div className="marginBottomSmall">
+              {this.state.labels.map((label, i) => (
+                <Tag
+                  key={"label" + i}
+                  size="md"
+                  type={labelColors[i % labelColors.length]}
+                >
+                  {label}
+                </Tag>
+              ))}
+            </div>
+          </Tile>
+        </IcseFormGroup>
+        <IcseFormGroup>
+          <Tile className="formInSubForm">
```

Comment:
add classname so that the width of these sub forms still matches the form width
 
<img width="619" alt="Screenshot 2023-08-31 at 11 13 58 AM" src="https://github.com/IBM/icse-react-assets/assets/11401353/b8daebae-dc04-42dc-bd91-15a93d12a0b8">